README.md

```diff
@@ -1 +1,237 @@
-# Vocdoni SDK
\ No newline at end of file
+# Vocdoni SDK
+
+The Vocdoni SDK is the easiest way to use [our new API][Vocdoni API], allowing
+you to easily create new voting processes, fetch their information and vote them.
```

Comment:
Proposal: The Vocdoni SDK is a convenient way to interact with the Vocdoni Protocol through the new API, allowing anyone to create, manage and participate in voting processes and collective decision making.